src/apps/OfficeView.js

```diff
@@ -0,0 +1,9 @@
+import React, { useEffect } from "react";
+import { useParams } from "react-router-dom";
+
+
+export default function() {
+    const { officeId } = useParams();
+
+    return <p>Hello and welcome to {officeId}</p>;
+}
```

Comment:
I guess I've never seen a functional React component written this way, but it works just the same 🤷‍♀️ Interesting! 👍